script.js

```diff
@@ -938,3 +938,621 @@ if ($(window).width() <= 700) {
 $(function(){
    $.ui.dialog.prototype._focusTabbable = function(){};
 });
+
```

Comment:
I'd vote to put the specific upload code into its own file; and load it only when required